### DISCUSSION TITLE
Gatsby Cloud build error 137

### DISCUSSION THREAD

**Discussion by hk86** [gatsbyjs/gatsby#30782] [Help] [answered]
Hi there,

since yesterday I'm not able to build on Gatsby Cloud. Everything seems to run as always, but during page queries are run I get:

```
11:23:37 AM:
/usr/local/bin/runner-command: line 25:  4051 Killed                  ${GATSBY} build --no-color
11:23:37 AM:
ERROR Failed to compile: Error: Exited with code 137
```

137 seems to be an out of memory error. Locally everything works fine, I can build on Netlify as well.

I use Gatsby v3. The only change I commited was to add a `process.env.GATSBY_VAR` on my client side code in one of my components.

Any ideas what might cause the issue? Was there a change on Gatsby Cloud?

Best regards,
HK

**Comment by LekoArts** [comment, marked as answer]
For Gatsby Cloud support please reach out here: https://www.gatsbyjs.com/support/
Thanks!

**Reply by hk86** [reply]
Thanks

**Comment by gentksb** [comment]
I'm facing the same problem. It maybe cause by GatsbyCloud free plan limit, but limits doesn't works well I think.

My v2 blog and v3 small website got the out-of-memory error.
Locally, `gatsby-plugin-inspecter` result is under 1GB Heap RAM.

![v3 web](https://user-images.githubusercontent.com/36843875/114403201-89645b80-9bdf-11eb-8088-f9648a1c5d5f.jpg)

![v2 blog](https://user-images.githubusercontent.com/36843875/114403410-b87acd00-9bdf-11eb-8935-d4d126645e1c.png)
The only thing we can do is to inform GatsbyCloud support.

**Reply by gentksb** [reply]
Today, my lighter website completes build successfully. But another is not.

**Reply by hk86** [reply]
There is a 1GB memory limit on the free plan. This is causing the issue for me.

**Reply by hk86** [reply]
The must have changed it recently.

**Comment by fnebenfuehr** [comment]
We are having the same issue since we upgraded to gatsbyjs 3.0. Switching to the performance plan does fix it temporarily.

**Comment by daydream05** [comment]
Running into the same issue. It'd be nice if there's a guide to find out which causes 1gb of RAM usage for gatsby builds and what we can do to lower it.
